python/tk_multi_loader/loader_action_manager.py

```diff
@@ -265,7 +264,47 @@ def has_actions(self, publish_type):
         my_mappings = mappings.get(publish_type, [])
         
         return len(my_mappings) > 0
-        
+
+    def _get_actions_for_folder(self, sg_data):
+        """
+        Retrieves the list of actions for a given folder.
+
+        :param sg_data: Publish to retrieve actions for
+        :return: List of actions.
+        """
+
+        publish_type = sg_data.get('type', None)
+
+        # check if we have logic configured to handle this publish type.
+        mappings = self._app.get_setting("entity_mappings")
+
+        # returns a structure on the form
+        # { "Shot": ["reference", "import"] }
+        actions = mappings.get(publish_type, [])
+
+        if len(actions) == 0:
+            return []
+
+        # convert created_at unix time stamp to shotgun time stamp
+        unix_timestamp = sg_data.get("created_at")
```

Comment:
Could you refactor this timestamp fix bit into its own method and update the other place in the file that uses this? Thanks!